pages/version-policy/index.md

```diff
@@ -39,17 +38,14 @@ Thus, upon the release of 2.0.0 where 1.10.4 and 1.9.7 are available:
 * **Minor Version**: Upon the availability of a new DC/OS Minor version 1.11.0, minor versions prior to and including 1.8.0 will no longer be supported as (N-2) includes 1.11 (N), 1.10 (N-1) and 1.9 (N-2).
 * **Maintenance Version**: Upon the availability of DC/OS Maintenance version 1.10.5, Maintenance versions prior to and including 1.10.0 will no longer be supported as (N-4) includes 1.10.5 (N), 1.10.4 (N-1), 1.10.3 (N-2), 1.10.2 (N-3) and 1.10.1 (N-4).<br>
 
-
 ## DC/OS Platform Version Compatibility Matrix
 
 The following matrix shows the platform components and operating environments on which DC/OS has been tested; DC/OS will run on these components. Customers running DC/OS on non-supported platform components should upgrade to a supported component. For clarity, Mesosphere only provides support services to paying customers under a written agreement; the term “supported” in Mesosphere documentation refers to whether the indicated software component has been tested for compatibility.
```

Comment:
We should include warnings about the fact that if customers are using a RHEL or CentOS then Mesosphere Kubernetes Engine requires CentOS/RHEL 7.5+ due to a well known kernel bug in that affects Kubernetes on all older operating systems. I am having trouble finding in the docs where we have put this warning before.

Reply:
This warning already exists in the fully rendered page.